dynawo/sources/Models/Modelica/PreassembledModels/HVDCStandard.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+    Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+    See AUTHORS.txt
+    All rights reserved.
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, you can obtain one at http://mozilla.org/MPL/2.0/.
+    SPDX-License-Identifier: MPL-2.0
+
+    This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+    simulation tool for power systems.
+-->
+<dyn:dynamicModelsArchitecture xmlns:dyn="http://www.rte-france.com/dynawo">
+  <dyn:modelicaModel id="HVDCStandard">
+    <dyn:unitDynamicModel id="hvdcstandard" name="Dynawo.Electrical.HVDC.Standard.HVDCStandard" initName ="Dynawo.Electrical.HVDC.Standard.HVDCStandard_INIT"/>
```

Comment:
I would have called it only hvdc. This would simplify the files later on.

Reply:
Fixed